Utils/Batcher/Unit1.pas

```diff
@@ -42,20 +66,47 @@   TForm1 = class(TForm)
 
 implementation
 uses
-  KM_Campaigns, KM_MissionScript;
+  KM_Campaigns, KM_Game, KM_MissionScript_Standard;
+
+const  
+  DefaultKaMOriginalTeamColors: array [0..MAX_HANDS-1] of Cardinal = (
+    $FF0707FF, //Red
+    $FFE3BB5B, //Cyan
+    $FF27A700, //Green
+    $FFFF67FF, //Magenta
+    $FF07FFFF, //Yellow
+    $FF577B7B, //Grey
+    $FF2383FB, //Orange
+    $FFFF0707, //Blue
+    $FF0BE73F, //Light green
+    $FF720468, //Purple
+    $FFFFFFFF, //White
+    $FF000000  //Black

```

Comment:
Looks like you copied your changed version by mistake. The original KaM default colours had black repeated.